MetamodelUML.md

```diff
@@ -747,6 +755,10 @@ De data-elementen worden naar de volgende aspecten gespecificeerd:
 | **- Formeel patroon**  | 0..1    | Algemeen metagegeven. |                                                                 |      | *Tagged value*    |     |
 | **Kardinaliteit**      | 1       | Algemeen metagegeven. | *lowerValue en upperValue van de metaclass MultiplicityElement* |      | *Multiplicity*    |     |
 | heeft **datatype**     | 1       | Binding aan een datatype.  | *datatype* = UML-datatype       |      | *type* = datatype |     |
+| **Minimumwaarde inclusief** | 0..1 | Algemeen metagegeven. Een data element mag maar één voorkomen van metagegeven **Minimumwaarde inclusief** of **Minimumwaarde exclusief** hebben. | | | _Tagged value_ | |
```

Comment:
Het is mooier om bij minimumwaarde _exclusief_ het niet over minimum waarde _inclusief_ te hebben. Dus:


| **Minimumwaarde exclusief** | 0..1 | Algemeen metagegeven. Een data element mag maar één voorkomen van metagegeven **Minimumwaarde exclusief** hebben. | | | _Tagged value_ | |

Reply:
Het punt is dat je niet zowel inclusief als exclusief kunt specificeren voor minimumwaarde. Hoewel de multipliciteit dat wel toestaat.

Reply:
Gezien dit een toelichting betreft en het volgens mij wel van belang is om deze extra verduidelijking te geven laat ik het liever  toch zo staan.

Reply:
Ahh, zo had  ik het niet gelezen. Misschien duidelijker (hoeft niet nu) "Een data element mag of een metagegeven **Minimumwaarde inclusief** of **Minimumwaarde exclusief** hebben, niet beide."

Reply:
Ja dat is duidelijker, dat pas ik nog even aan, thanks!